asm/bss_8001C4D0.s

```diff
@@ -2121,7 +2121,7 @@ glabel D_8018EDFB
 glabel D_8018EDFC
 .skip 4
 
-glabel gStartMenuCounterForDemos
+glabel gMenuTimingCounter
```

Comment:
I realized this is also used in other menus. For certain menus, once this reaches a certain value or multiple of a value, it will play an audio clip prompting the player to make a selection, hit ok, etc.